<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GlobalShortcutsDialog</class>
 <widget class="QDialog" name="GlobalShortcutsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure Shortcuts</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../data/data.qrc">
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="gnome_container" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="gnome_checkbox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Use Gnome's shortcut keys</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="gnome_open">
        <property name="text">
         <string>Open...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="mac_container" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="mac_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>You need to launch System Preferences and turn on &quot;&lt;span style=&quot; font-style:italic;&quot;&gt;Enable access for assistive devices&lt;/span&gt;&quot; to use global shortcuts in Clementine.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mac_open">
        <property name="text">
         <string>Open...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="list">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Shortcut</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="shortcut_options">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Shortcut for %1</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="radio_none">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>&amp;None</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radio_default">
        <property name="text">
         <string>De&amp;fault</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radio_custom">
        <property name="text">
         <string>&amp;Custom</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="change">
        <property name="text">
         <string>Change shortcut...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>list</tabstop>
  <tabstop>radio_none</tabstop>
  <tabstop>radio_default</tabstop>
  <tabstop>radio_custom</tabstop>
  <tabstop>change</tabstop>
  <tabstop>button_box</tabstop>
 </tabstops>
 <resources>
  <include location="../../data/data.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>GlobalShortcutsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>GlobalShortcutsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>436</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gnome_checkbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>list</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>82</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gnome_checkbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>shortcut_options</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
